### ISSUE TITLE
d and D do not close the last tab

### ISSUE THREAD

**Issue by njsg** [5digits/dactyl#31]
d and D work as expected except when there is only one tab in the window. If there is only one tab, d/D will not close that tab at all. Instead of bringing up a blank tab, the current tab stays open.

Pentadactyl 7166
Firefox 31.5.3

**Comment by njsg** [comment]
On another computer, I still had Pentadactyl 7123 with Firefox 31.5.3. I tried d and D, which were working as expected. Then I upgraded to Pentadactyl 7166 and that broke it.

**Comment by polyzen** [comment]
@njsg, are you still having this issue? If not, please close this.

**Comment by noctuid** [comment]
This problem never went away for me (unless there was a fix in the past month or so).

**Comment by polyzen** [comment]
@noctuid, works here on penta master and ff 46.

**Comment by njsg** [comment]
Here, the problem went away. Unfortunately, I did not write down the details.

IIRC, it went away after I removed some tab-related pentadactyl plugin ("tab options"?). So, that would mean some version of the plugin was incompatible (or at least behaved differently) with the newer version of pentadactyl. I'm currently running hg7284, 2016/04/07, and now I don't have this problem anymore.

noctuid: which version of pentadactyl, any (pentadactyl) plugins?

**Comment by noctuid** [comment]
My version of pentadactyl is newer than that (2016/04/29). I am using the tab options plugin. It's an official plugin though; was it updated any time recently?

**Comment by noctuid** [comment]
Yes, the tab-options plugin is the culprit. I think this should be left open since it's an official plugin though.

**Comment by polyzen** [comment]
Appears to be a duplicate of #51 and #4

**Comment by noctuid** [comment]
If it's not a duplicate (it's not clear from those issues if `:tabclose` was working at all), the fix is the same. I's unfortunate it was never added to the plugin.
